common/lib/xmodule/xmodule/modulestore/tests/test_assetstore.py

```diff
@@ -124,7 +124,7 @@ def _compare_metadata(self, mdata1, mdata2, msg=None):
         if type(mdata1) != type(mdata2):  # lint-amnesty, pylint: disable=unidiomatic-typecheck
             self.fail(self._formatMessage(msg, u"{} is not same type as {}".format(mdata1, mdata2)))
         for attr in mdata1.ATTRS_ALLOWED_TO_UPDATE:  # lint-amnesty, pylint: disable=redefined-outer-name
-            assert getattr(mdata1, attr) == getattr(mdata2, attr), msg
```

Comment:
@ormsbee @kdmccormick Most of the test failures are because of the differences in unittest and pytest differences. For now in these cases I have reverted to the older code from https://github.com/edx/edx-platform/pull/26530 so that we can get master green.

Reply:
Okay, figured out why. There are [custom equality checkers](https://github.com/edx/edx-platform/blob/a332aac1f6a04505391fea1c785be1d24ab46d93/common/lib/xmodule/xmodule/modulestore/tests/test_mixed_modulestore.py#L151) being added in this test class. Updating the tests.